Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/MagnitudeSpectrum.cpp

```diff
@@ -0,0 +1,20 @@
+#include "MagnitudeSpectrum.hpp" 
+
+MagnitudeSpectrum::MagnitudeSpectrum(int iBufferSize_, int iSampleRate_) 
+   : SpectrumBase(iBufferSize_, iSampleRate_)
+{
+
+   return; 
+}
+
+
+void MagnitudeSpectrum::CalculateSpectrum()
+{
+   
+   // We can just interleave the data,
```

Comment:
um ecuse me muiltipleline coments shoud be likke
```c++
/*
 * i have aids
 * please help me
 */
```

Reply:
Please refrain from using such language on the public GitHub repo.